.github/workflows/build-crates-individually.yml

```diff
@@ -1,5 +1,11 @@
 name: Build crates individually
 
+# Ensures that only a single CI workflow runs at a time in Pull Requests. The
```

Comment:
This isn't a full sync on `main` or a Google Cloud job, so its concurrency is automatically limited to the correct limit for GitHub actions.

Reply:
But maybe it would be less confusing (and faster) if it works the same for GitHub actions and Google Cloud jobs.